config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -194,6 +194,7 @@ periodics:
   cron: 0 8-20/12 * * *
   labels:
     preset-e2e-scalability-common: "true"
+    preset-e2e-scalability-periodics: "true"
```

Comment:
Similarly as in other PRs - can we split release branches from the master?

Reply:
It is already split (at the PR level), please check individual commits here.